src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -50,9 +52,9 @@
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Major;
 import seedu.address.model.person.Name;
-import seedu.address.model.person.Tutorial;
 import seedu.address.model.person.Year;
 import seedu.address.model.socialmedialink.SocialMediaLink;
+import seedu.address.model.tutorial.Tutorial;
 import seedu.address.testutil.EditPersonDescriptorBuilder;
 
 public class EditCommandParserTest {
```

Comment:
update the method `parse_invalidValue_failure` with invalid tutorial, update `parse_allFieldsSpecified_success`, update `parse_oneFieldSpecified_sucesss`